account-number-protector/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <script>
+    function accountNumberProtector(cardNumberString) {
+      let lastFour = '';
+      for (let i = 15; i > 11; i--) {
+        if (i > 11) {
+          lastFour = cardNumberString[i] + lastFour;
+        }
```

Comment:
This gets them in reverse, and then concatenates them backwards?

Reply:
i guess im confused on how this works, but it still returns the expected value

Reply:
Yeah it starts from the end and works its way back. Each time concatenating the cardNumberString at i onto the front of the string.